src/app/app-routing.module.ts

```diff
@@ -4,13 +4,19 @@ import { ProjectListComponent } from './projects/project-list/project-list.compo
 import { ProjectDetailsComponent } from './projects/project-details/project-details.component';
 import { BuildDetailsComponent } from './builds/build-details/build-details.component';
 import { LicensesComponent } from './licenses/licenses.component';
+import { LoginComponent } from './auth/login/login.component';
+import { UnauthorizedComponent } from './auth/unauthorized/unauthorized.component';
+import { ForbiddenComponent } from './auth/forbidden/forbidden.component';
 
 
 const routes: Routes = [
   { path: '', component: ProjectListComponent },
   { path: 'project/:projectId', component: ProjectDetailsComponent },
   { path: 'build/:projectId/:buildId', component: BuildDetailsComponent },
   { path: 'third-party-licenses', component: LicensesComponent },
+  { path: 'login', component: LoginComponent },
+  { path: 'unauthorized', component: UnauthorizedComponent},
+  { path: 'forbidden', component: ForbiddenComponent},
```

Comment:
Maybe we want to group these auth-related endpoints inside `/auth/...`? I.e. changing `/login` to `/auth/login`, etc.

I see no chance in us having any path collisions with any of these new routes, so it's kind of only for the aestetic, and potentially path autocompletion in the browser. Neither are really that good of an argument, so I guess you could categorize this comment as a "nitpick"

Reply:
Old discussion, dont remember. Resolving for now.